src/README.md

```diff
@@ -543,7 +543,7 @@ Source:
 ### dpsl.diagnostics.hardwareButton.*
 | Function Name | Definition | Permission needed to access | Released in `dpsl` version |
 ------------ | ------------- | ------------- | ------------- |
-| runPowerButtonRoutine | () => Promise\<Routine\> | `os.diagnostics` | 1.3.6 |
+| runPowerButtonRoutine | (params: PowerButtonRoutineParams) => Promise\<Routine\> | `os.diagnostics` | 1.3.6 |
```

Comment:
I feel you also need to prepare the struct of this one? For example, I saw there is a `AcPowerRoutineParams` section to describe how the parameters look like.

Reply:
Ah sorry, my local is outdated.